flexmeasures/ui/templates/base.html

```diff
@@ -11,6 +11,9 @@
     </title>
     {% endblock head %}
     <meta charset="windows-1252">
+    {% if FLEXMEASURES_ENFORCE_SECURE_CONTENT_POLICY == true %}
```

Comment:
Here you check for "true" (no capital T). I guess this win't work anymore, please check! (maybe Python `True` becomes Javascript  `true`)

Is it safer not to do a direct comparison?

Reply:
It did work, but i will confirm it?

what do you mean by direct comparison?

Reply:
By testing without comparison I mean to just check `if FLEXMEASURES_ENFORCE_SECURE_CONTENT_POLICY`

Reply:
What does the :+1: mean in response to my question? 

And what do you mean with "but i will confirm it"?

Reply:
- About "but i will confirm it", I meant that I will check if Python `True` becomes Javascript `true`. So I just confirmed it and it becomes Javascript `true`.
- Direct comparison also works.

Reply:
Thanks! Then you could resolve this :)  (I'll do it now)